### ISSUE TITLE
'isbot' import error on basic installation instructions prevents running the basic starter project

### ISSUE THREAD

**Issue by trooperandz** [remix-run/remix#8428]
### Reproduction

1) Follow the basic installation instructions per the documentation.
2) Run the final step command from the root project directory: `npx remix build`
3) Note the error returned: `SyntaxError: The requested module 'isbot' does not provide an export named 'default'`

### System Info

```shell
System:
    OS: macOS 14.1.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 701.89 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.8.1 - ~/.nvm/versions/node/v20.8.1/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 10.1.0 - ~/.nvm/versions/node/v20.8.1/bin/npm
    Watchman: 2023.02.20.00 - /usr/local/bin/watchman
  Browsers:
    Chrome: 120.0.6099.129
    Safari: 17.1
  npmPackages:
    @remix-run/dev: ^2.4.1 => 2.4.1
    @remix-run/node: ^2.4.1 => 2.4.1
    @remix-run/react: ^2.4.1 => 2.4.1
    @remix-run/serve: ^2.4.1 => 2.4.1
```

### Used Package Manager

npm

### Expected Behavior

The starter project app should execute the serve command successfully with no error.

### Actual Behavior

The following error is returned:
`SyntaxError: The requested module 'isbot' does not provide an export named 'default'`

**Comment by jstafman** [comment]
Check the version of `isbot` listed in your `package.json` file. If it is >=4.0.0, it's because `isbot` removed the default export in favor of a named export. Until remix catches up, the easiest thing to do is downgrade to the latest v3, which is currently 3.8.0. You can do this by either updating the version in your `package.json` file and running `npm install` or running `npm install isbot@3.8.0`.

<img width="893" alt="image" src="https://github.com/remix-run/remix/assets/2019325/92f4b898-5ff8-4ac2-ae0c-dd00b2b2fb23">

[Source](https://github.com/omrilotan/isbot)

**Comment by Vinorcola** [comment]
It could be nice to update the tutorial that way:

```diff
- npm i @remix-run/node @remix-run/react @remix-run/serve isbot react react-dom
+ npm i @remix-run/node @remix-run/react @remix-run/serve isbot@3.8.0 react react-dom
```

**Comment by Vinorcola** [comment]
Looks like a fix is already shipped for the next Remix version: https://github.com/remix-run/remix/pull/8415

**Comment by brophdawg11** [comment]
Closing as a duplicate of #8407 and #8392.  As stated, this will be fixed by #8415 in the next Remix version.

For now, just `npm install isbot@3` to pin to v3 in your `package.json` 👍
